pages/common/git-checkout.md

```diff
@@ -15,6 +15,10 @@
 
 `git checkout {{branch_name}}`
 
+- Switch to the branch previously checked out:
```

Comment:
Switch to the previously checked out branch ?

In the other examples, "branch" almost always comes at the end.

Reply:
Fixed.